Gems/PhysX/Code/Source/RigidBodyComponent.cpp

```diff
@@ -404,6 +404,8 @@ namespace PhysX
         m_interpolator = std::make_unique<TransformForwardTimeInterpolator>();
         m_interpolator->Reset(transform.GetTranslation(), rotation);
 
+        GetEntity()->GetTransform()->SetOnParentChangedBehavior(AZ::OnParentChangedBehavior::DoNotUpdate);
```

Comment:
suggestion: A comment here to indicate _why_ we're doing this would be helpful for context

Reply:
Yeah, good idea, will update.